src/main/webapp/profileBuilder.html

```diff
@@ -5,11 +5,137 @@
+                            <input type="text" name="major-input" class="form-control"  placeholder="Major">
+                        </label>
+                    </div>
+
+                        <input type="submit" value="Submit">
+                    </div>
+
+                <!-- Mentor Input Form -->
+                <div id="Mentor" style="display: none">
+                <!-- full name -->
+                    <div>
+                        <label style="color:black;" name="profName-input"> Full Name:
+                            <input type="text" name="profName-input" class="form-control">
+                        </label>
+                    </div>
+                    <!-- Company input -->
+                    <div>
+                        <label style="color:black;" name="company-input"> Company:
+                            <input type="text" name="company-input" class="form-control" placeholder="Company">
+                        </label>
+                    </div>
+                    <!-- Career Title input -->
+                    <div>
+                        <label style="color:black;" name="careerTitle-input"> Career Title:
+                            <input type="text" name="careerTitle-input" class="form-control"  placeholder="Career Title">
+                        </label>
+                    </div>
+                    
+                    <!-- Specialty Input  -->
+                    <label style= "color: black;"> User Type:
+                        <br>
+                        <select placeholder={{specialty}} name="specialty-Input" multiple>
+                            <option value="Machine Learning">Maching Learning</option>
+                            <option value="Artificial Intelligence">Artificial Intelligence</option>
+                            <option value="Systems">Systems</option>
+                            <option value="Networks">Networks</option>
+                            <option value="Security">Security</option>
+                            <option value="Database">Database</option>
+                            <option value="Graphics">Graphics</option>
+                            <option value="Electrical Engineering">Electrical Engineering</option>
+                            <option value="DoS">DoS</option>
+                        </select>
+                    </label>
+                    <p style= "color: black;">Hold down the Ctrl (windows) or Command (Mac) button to select multiple options.</p>
+
+                    <input type="submit" value="Submit">
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- Footer -->
+    <footer class="py-5 bg-black">
+      <div class="container">
+        <p class="m-0 text-center text-white small">Copyright &copy; Techish 2020</p>
+      </div>
+      <!-- /.container -->
+    </footer>
+    <!-- Bootstrap core JavaScript -->
+    <script src="vendor/jquery/jquery.min.js"></script>
```

Comment:
Includes in head/top?

Reply:
What do you mean by this, what line specifically?

Reply:
Done